game/operation/operation.py

```diff
@@ -181,8 +182,8 @@ def notify_info_generators(
                 for awacs in airsupportgen.air_support.awacs:
                     gen.add_awacs(awacs)
 
-            for aewc in airgen.CustomAirSupport.aewc:
-                gen.add_aewc(aewc)
+            for aewc in airsupportgen.air_support.awacs:
+                gen.add_awacs(aewc)
```

Comment:
This will create duplicates when `cls.player_awacs_enabled` is True. Just remove the if?